docs/components/modeler/desktop-modeler/connect-to-camunda-8.md

```diff
@@ -10,7 +10,7 @@ Desktop Modeler can directly deploy diagrams and start process instances in Camu
 
 ![deployment icon](./img/deploy-icon.png)
 
-2. Click **Camunda Platform 8 SaaS**:
+1. Click **Camunda 8 SaaS**:
```

Comment:
Will this button name change happen for 8.3? Can you confirm with the Desktop Modeler team?

Reply:
Yes 👍 Confirmed with @smbea